src/test/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatterSpec.scala

```diff
@@ -15,6 +17,28 @@ class TSVFormatterSpec extends FreeSpec with ScalaFutures with Matchers with Ins
 
   "TSVFormatter" - {
 
+    "Sparse data fields should pass for" - {
```

Comment:
Is this the test that would have failed had it existed previously, and now does not fail?